batch-nft-reveal/test/NFTCollection.test.ts

```diff
@@ -185,7 +185,7 @@ describe("NFTCollection", async function () {
         "./test/data/revealed_metadata.txt",
         "utf8"
       );
-      const uri = await nftCollection.tokenURI(0);
+      const uri = await nftCollection.tokenURI(1);
```

Comment:
Why the other test `should show unrevealed message` is querying token with id `0`?

Reply:
When you `tokenUri` nonexistent token it shows the unrevealed message. We need to add a `require` when you ask for nonexistent token's tokenUri.